### ISSUE TITLE
Exporting Go annotation per GO domain/GO term

### ISSUE THREAD

**Issue by samehmgd** [compomics/peptide-shaker#288]
Hi 
 I would like to draw a pie explaining the percentage of each subcellular compartment (mitochondrial, ribosomal ,,,etc) from the GO. the point here that i want to export (for example) a list of all proteins with their GO terms and GO domain. 
To be more representing, the header is
protein accession, GO term,GO domain, log2 diff, freq all, freq dataset 

what i see here that i can export a list of protein for a certain GO term not for all GO domain.

Thanks for your help in advaance

**Comment by hbarsnes** [comment]
Hi Sameh,

The GO Analysis tab in PeptideShaker uses so-called Gene Ontology Enrichment Analysis (GOEA). GOEA analyzes the frequencies of GO terms in your dataset and compares these to the frequencies of the same GO terms in the species specific version of Ensembl (www.ensembl.org). In order to not get too many terms the GOSlim UniProtKB-GOA (www.ebi.ac.uk/GOA) is used. This shows you if certain GO terms are found more or less often in your dataset compared to the distribution of the same terms in Ensembl.

In other words, the GO Analysis tab is not really designed to give you the information you require for the kind of analysis you want to do. You would therefore have to export the protein accession numbers, potentially with the GO terms via a custom protein report (Export > Identification Features), from PeptideShaker and perform the GO analysis outside of PeptideShaker.

Best regards,
Harald

**Comment by samehmgd** [comment]
Thanks for your reply. I believe this is an exporting feature for the software. I hope you could consider including it in the future.
